underfs/swift/pom.xml

```diff
@@ -39,36 +39,6 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.hadoop</groupId>
```

Comment:
@calvinjia yes, we don't need this Hadoop stuff in the swift module. We use JOSS so all those Hadoop dependencies are not needed in the Swift module.